conans/test/unittests/client/cmd/export/test_update_revision_in_metadata.py

```diff
@@ -18,8 +17,9 @@ class UpdateRevisionInMetadataTests(unittest.TestCase):
 
     def setUp(self):
         ref = ConanFileReference.loads("lib/version@user/channel")
-        self.package_layout = PackageCacheLayout(base_folder=temp_folder(), ref=ref,
-                                                 short_paths=False, no_lock=True)
+        # FIXME: 2.0: PackageCacheLayout does not exist anymore
```

Comment:
I think we can remote all these tests, maybe adding a TODO to add some unittests for the cache on getting the fields that replace the metadata for 2.0.

Reply:
Yeah, but I think it's better to keep them for now and, open another PR with those new tests and get rid of these ones. Anyway, I'm gonna add that TODO